### PR TITLE
button: create button with default values

### PR DESCRIPTION
**Pull request by carizzuti**
Created a button component using default values for title and isDisabled

### CODE REVIEW

**Comment by carizzuti** [comment]
I have updated this pull request with additional functionality

**Review by Nap1400 — APPROVED** [review]

File: src/components/button/Button.styles.js [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+        backgroundColor: '#7199FF',
+        borderRadius: 8,
+        height: 60,
+        width: 319,
+        justifyContent: 'center',
+    },
+    buttonDisabled: {
+        backgroundColor: '#D1D1D1',
+    },
+    container: {
```
[the remark this answers is not in the document]
Comment (raynesmike) [reply src/components/button/Button.styles.js:14]: We can clean it later if we dont need the container style.

**Review by raynesmike — APPROVED** [review]
I approve this Button addition

File: src/components/button/Button.styles.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (carizzuti) [reply src/components/button/Button.styles.js:14]: It's not used for anything looks like I just didn't remove something I was testing

**Comment by carizzuti** [comment]
image

File: src/components/button/Button.styles.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+    buttonEnabled: {
```
Comment (lasharela) [inline src/components/button/Button.styles.js:4]: This is general styles for button. Try to separate `buttonContainer` and then you can do `buttonDisable` and `buttonEnable` styles.

File: src/components/button/Button.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+import { styles } from './Button.styles';
+
+export const Button = ({title = 'Button', isDisabled = false, ...rest}) => {
+    let buttonStyle;
+
+    if (isDisabled) {
+        buttonStyle = [styles.buttonEnabled, styles.buttonDisabled];
```
Comment (lasharela) [inline src/components/button/Button.js:9]: you can improve this. for enable set only enabled, for disabled disabled.

File: src/components/button/Button.styles.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        backgroundColor: '#7199FF',
+        borderRadius: 8,
+        height: 60,
+        width: 319,
```
Comment (lasharela) [inline src/components/button/Button.styles.js:8]: Don't use fixed width

**Review by lasharela — APPROVED** [review]

File: src/components/button/Button.styles.js (hunk 2 above)
Comment (carizzuti) [reply src/components/button/Button.styles.js]: I've added a general `buttonContainer` style

File: src/components/button/Button.js (hunk 3 above)
Comment (carizzuti) [reply src/components/button/Button.js]: I have changed this to use `buttonContainer` with the difference being applied with `buttonDisabled` and `buttonEnabled`

File: src/components/button/Button.styles.js (hunk 4 above)
Comment (carizzuti) [reply src/components/button/Button.styles.js]: This has been changed to use percentage dimensions

**Review by leo-he-1 — APPROVED** [review]
Component looks good. It helped me with mine too.

**Review by mshago — APPROVED** [review]
Looks great

**Pull request merged**